docs/governance/governance.md

```diff
@@ -0,0 +1,241 @@
+
+As with all decisions the decision to add a Team member is subject to article 3.4.
+
+### 5.2 Removing Team Members
+
+Teams themselves decide the procedure to remove Team members. As for the procedure of adding Team Members, the procedure to remove Team Members should reflect the sensitivity of the position.
+
+As with all decisions, the decision to remove a Team member is subject to article 3.4.
+
+## 6 Steering committee
+
+### 6.1 Responsibilities
+
+The Steering Committee shall be responsible for:
+
+- Outlining the technical direction of the EESSI project and setting priorities
+- Conflict resolution: if there is a conflict between the people with the aforementioned roles, the steering committee shall mediate the dispute and make a final decision if consensus cannot be reached
+- The [EESSI Charter](charter.md)
+- The [EESSI Governance](governance.md)
+- The [EESSI Policies](policies.md)
+
+### 6.2 Members and Chairs
+
+The Steering Committee is made up of community members with sustained and recognized contributions over time. Members take part in the Steering Committee as private individuals (i.e. they do not represent their employer(s)).
+
+#### 6.2.1 Termination of membership
+
+Membership of the Steering Committee can terminate in three ways:
+
+- A Member can resign
+- A Member may be voted out. In this case, the vote needs to be unanimous between the other Members. This procedure is intended to provide a mechanism for addressing extraordinary circumstances where a member's behavior or actions are deemed severely detrimental to the committee's function or reputation, and/or to the EESSI project.
+- If a Member is inactive for 6 months and does not reply to communication from the rest of the Steering Committee, the Member may be voted out by the other Steering Committee members. In this case, [regular voting rules](#voting-by-sc) apply.
+
+#### 6.2.2 New members
+
+New Members must be recommended by one or more Steering Committee members.
+
+The Steering Committee will make the recommendation known to the community, and collect feedback on the recommendation.
+The Steering Committee then weighs the feedback, and votes on whether to accept the new member.
+
+In the selection process, the Steering Committee will consider the following:
+
+- The Steering Committee ideally consists of an odd number of Members (to reduce chances of a tie when voting);
+- No more than 30% of the Members should be employed by commercial entities (to avoid that the project becomes dominated by commercial interests);
+- The number of Members working for the same employer should be limited to 1 (to avoid that the project becomes dominated by a single employer, or a small group of employers);
```

Comment:
```suggestion
- The number of Members working for the same employer should be limited to 1 (to avoid that the project becomes dominated by a single company or institution, or a small group thereof);
```